client-go/clientset/versioned/clientset.go

```diff
@@ -21,23 +21,23 @@ import (
 	discovery "k8s.io/client-go/discovery"
 	rest "k8s.io/client-go/rest"
 	flowcontrol "k8s.io/client-go/util/flowcontrol"
-	jobsetv1alpha1 "sigs.k8s.io/jobset/client-go/clientset/versioned/typed/jobset/v1alpha1"
+	jobsetv1alpha2 "sigs.k8s.io/jobset/client-go/clientset/versioned/typed/jobset/v1alpha2"
```

Comment:
were the client-go files automatically moved and updated?

Reply:
I moved them - was I not supposed to?

Reply:
@danielvegamyhre those were supposed to be auto-generated, right?

@vsoch I assume they should get generated automatically via `make generate`

Reply:
files in client-go/ are auto-generated by hack/update-codegen.sh so i think we need to update that script to use the v1alpha2 package instead of v1alpha1.

Reply:
I did see the docs here: https://book.kubebuilder.io/multiversion-tutorial/tutorial.html but I found it confusing at best. It seems to walk through using the tool to generate a second version, and I didn't think we wanted another directory, but rather just a rename (e..g., the MPI operator just has their latest: https://github.com/kubeflow/mpi-operator/tree/master/pkg/apis/kubeflow). Feel free to close this and just have someone else do it if I'm totally missing the boat.